[2024-03-06T02:22:28.273+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Extraçao_de_dados_do_linkedin.Teste manual__2024-03-06T02:22:25.908547+00:00 [queued]>
[2024-03-06T02:22:28.285+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Extraçao_de_dados_do_linkedin.Teste manual__2024-03-06T02:22:25.908547+00:00 [queued]>
[2024-03-06T02:22:28.285+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-03-06T02:22:28.329+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): Teste> on 2024-03-06 02:22:25.908547+00:00
[2024-03-06T02:22:28.336+0000] {standard_task_runner.py:60} INFO - Started process 736 to run task
[2024-03-06T02:22:28.338+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Extraçao_de_dados_do_linkedin', 'Teste', 'manual__2024-03-06T02:22:25.908547+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/dag_exemplo.py', '--cfg-path', '/tmp/tmpjm5xqq_f']
[2024-03-06T02:22:28.341+0000] {standard_task_runner.py:88} INFO - Job 46: Subtask Teste
[2024-03-06T02:22:28.408+0000] {task_command.py:423} INFO - Running <TaskInstance: Extraçao_de_dados_do_linkedin.Teste manual__2024-03-06T02:22:25.908547+00:00 [running]> on host 43d6e1a41402
[2024-03-06T02:22:29.324+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Extraçao_de_dados_do_linkedin' AIRFLOW_CTX_TASK_ID='Teste' AIRFLOW_CTX_EXECUTION_DATE='2024-03-06T02:22:25.908547+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-06T02:22:25.908547+00:00'
[2024-03-06T02:22:29.326+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-06T02:22:29.328+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'pip install selenium']
[2024-03-06T02:22:29.344+0000] {subprocess.py:86} INFO - Output:
[2024-03-06T02:22:31.544+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2024-03-06T02:22:32.630+0000] {subprocess.py:93} INFO - Collecting selenium
[2024-03-06T02:22:32.750+0000] {subprocess.py:93} INFO -   Downloading selenium-4.18.1-py3-none-any.whl.metadata (6.9 kB)
[2024-03-06T02:22:32.980+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.26 in /home/***/.local/lib/python3.8/site-packages (from urllib3[socks]<3,>=1.26->selenium) (1.26.18)
[2024-03-06T02:22:33.294+0000] {subprocess.py:93} INFO - Collecting trio~=0.17 (from selenium)
[2024-03-06T02:22:33.318+0000] {subprocess.py:93} INFO -   Downloading trio-0.24.0-py3-none-any.whl.metadata (4.9 kB)
[2024-03-06T02:22:33.843+0000] {subprocess.py:93} INFO - Collecting trio-websocket~=0.9 (from selenium)
[2024-03-06T02:22:33.865+0000] {subprocess.py:93} INFO -   Downloading trio_websocket-0.11.1-py3-none-any.whl.metadata (4.7 kB)
[2024-03-06T02:22:34.041+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2021.10.8 in /home/***/.local/lib/python3.8/site-packages (from selenium) (2023.11.17)
[2024-03-06T02:22:34.042+0000] {subprocess.py:93} INFO - Requirement already satisfied: typing_extensions>=4.9.0 in /home/***/.local/lib/python3.8/site-packages (from selenium) (4.9.0)
[2024-03-06T02:22:34.051+0000] {subprocess.py:93} INFO - Requirement already satisfied: attrs>=20.1.0 in /home/***/.local/lib/python3.8/site-packages (from trio~=0.17->selenium) (23.2.0)
[2024-03-06T02:22:34.052+0000] {subprocess.py:93} INFO - Requirement already satisfied: sortedcontainers in /home/***/.local/lib/python3.8/site-packages (from trio~=0.17->selenium) (2.4.0)
[2024-03-06T02:22:34.052+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna in /home/***/.local/lib/python3.8/site-packages (from trio~=0.17->selenium) (3.6)
[2024-03-06T02:22:34.240+0000] {subprocess.py:93} INFO - Collecting outcome (from trio~=0.17->selenium)
[2024-03-06T02:22:34.259+0000] {subprocess.py:93} INFO -   Downloading outcome-1.3.0.post0-py2.py3-none-any.whl.metadata (2.6 kB)
[2024-03-06T02:22:34.476+0000] {subprocess.py:93} INFO - Requirement already satisfied: sniffio>=1.3.0 in /home/***/.local/lib/python3.8/site-packages (from trio~=0.17->selenium) (1.3.0)
[2024-03-06T02:22:34.477+0000] {subprocess.py:93} INFO - Requirement already satisfied: exceptiongroup in /home/***/.local/lib/python3.8/site-packages (from trio~=0.17->selenium) (1.2.0)
[2024-03-06T02:22:34.818+0000] {subprocess.py:93} INFO - Collecting wsproto>=0.14 (from trio-websocket~=0.9->selenium)
[2024-03-06T02:22:34.837+0000] {subprocess.py:93} INFO -   Downloading wsproto-1.2.0-py3-none-any.whl.metadata (5.6 kB)
[2024-03-06T02:22:35.409+0000] {subprocess.py:93} INFO - Collecting PySocks!=1.5.7,<2.0,>=1.5.6 (from urllib3[socks]<3,>=1.26->selenium)
[2024-03-06T02:22:35.428+0000] {subprocess.py:93} INFO -   Downloading PySocks-1.7.1-py3-none-any.whl.metadata (13 kB)
[2024-03-06T02:22:35.595+0000] {subprocess.py:93} INFO - Requirement already satisfied: h11<1,>=0.9.0 in /home/***/.local/lib/python3.8/site-packages (from wsproto>=0.14->trio-websocket~=0.9->selenium) (0.14.0)
[2024-03-06T02:22:35.641+0000] {subprocess.py:93} INFO - Downloading selenium-4.18.1-py3-none-any.whl (10.0 MB)
[2024-03-06T02:22:37.256+0000] {subprocess.py:93} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 10.0/10.0 MB 6.2 MB/s eta 0:00:00
[2024-03-06T02:22:37.277+0000] {subprocess.py:93} INFO - Downloading trio-0.24.0-py3-none-any.whl (460 kB)
[2024-03-06T02:22:37.554+0000] {subprocess.py:93} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 460.2/460.2 kB 1.6 MB/s eta 0:00:00
[2024-03-06T02:22:37.573+0000] {subprocess.py:93} INFO - Downloading trio_websocket-0.11.1-py3-none-any.whl (17 kB)
[2024-03-06T02:22:37.748+0000] {subprocess.py:93} INFO - Downloading PySocks-1.7.1-py3-none-any.whl (16 kB)
[2024-03-06T02:22:37.916+0000] {subprocess.py:93} INFO - Downloading wsproto-1.2.0-py3-none-any.whl (24 kB)
[2024-03-06T02:22:38.293+0000] {subprocess.py:93} INFO - Downloading outcome-1.3.0.post0-py2.py3-none-any.whl (10 kB)
[2024-03-06T02:22:41.729+0000] {subprocess.py:93} INFO - Installing collected packages: wsproto, PySocks, outcome, trio, trio-websocket, selenium
[2024-03-06T02:22:44.807+0000] {subprocess.py:93} INFO - Successfully installed PySocks-1.7.1 outcome-1.3.0.post0 selenium-4.18.1 trio-0.24.0 trio-websocket-0.11.1 wsproto-1.2.0
[2024-03-06T02:22:44.896+0000] {subprocess.py:93} INFO - 
[2024-03-06T02:22:44.897+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.3.2 -> 24.0
[2024-03-06T02:22:44.898+0000] {subprocess.py:93} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2024-03-06T02:22:45.107+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-06T02:22:45.159+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=Extraçao_de_dados_do_linkedin, task_id=Teste, execution_date=20240306T022225, start_date=20240306T022228, end_date=20240306T022245
[2024-03-06T02:22:45.337+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-06T02:22:45.483+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
